src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -2024,10 +2186,6 @@ namespace WaterThermalTanks {
 
 				} // DO HPWaterHeaterNum = 1, NumHeatPumpWaterHeater
 
-				if ( ErrorsFound ) {
-					ShowFatalError( "Errors found in getting " + cCurrentModuleObject + " input. Preceding condition causes termination." );
-				}
-
```

Comment:
Moved all fatal errors up to manager. "Previous errors" should show which object has problems.